lib/pbench/cli/server/repair.py

```diff
@@ -61,7 +61,7 @@ def repair_audit(kwargs):
         didit = False
         for key, value in a.items():
             if type(value) is str and len(value) > LIMIT:
-                a[key] = value[:LIMIT]
+                a[key] = f"[TRUNC({len(value)})]" + value[:LIMIT]
```

Comment:
My desire is that, having run the repair utility once, running it again shouldn't report the same things needing repair.  I realize that that is a bit of a pipe dream, since, if a column is "too big", we don't really know how much to shorten the key values.  (Yeah, I sure that we could figure it out -- we're smart guys -- but I'm not sure it's really worth it).  But, truncating it to the limit and then _adding_ characters to it **guarantees** that it will be over the limit "next time".  But, I suppose, if we're gonna fail, we might as well fail as early as possible.  🤣